sonar-db/src/test/resources/org/sonar/db/duplication/DuplicationDaoTest/insert-result.xml

```diff
@@ -15,8 +15,7 @@
   <projects id="1" uuid="1" kee="foo" enabled="1" scope="FIL" qualifier="CLA"/>
 
   <duplications_index id="1"
-                      project_snapshot_id="1"
-                      snapshot_id="2"
+                      analysis_uuid="u1"
```

Comment:
I didn't use the same format in my commits. In my Intellij configuration attributes are not aligned. We should discuss about the convention.